packages/docusaurus-theme-openapi-docs/package.json

```diff
@@ -47,7 +47,7 @@
     "docusaurus-plugin-sass": "^0.2.3",
     "file-saver": "^2.0.5",
     "lodash": "^4.17.20",
-    "node-polyfill-webpack-plugin": "^2.0.1",
```

Comment:
This makes the plugin compatible with Rspack, see https://github.com/Richienb/node-polyfill-webpack-plugin/releases/tag/v3.0.0

3.0 is only this change, but baybe you could also upgrade to v4.0 although I'm not sure it's side-effect free